publish.html

```diff
@@ -629,6 +629,57 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO can be used as a public data repository with either a custom web app or using the OMERO.gallery app.</p>
```

Comment:
This is a bit misleading, because you can allow public access without having a custom front-end if you like. What about:
"OMERO can be used as a public data repository by adding a 'public user' and you can also customise the appearance of OMERO.web or use a separate custom app to tailor the front-end for your target audience."